modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,6 +1,100 @@
 = Release Notes for Couchbase Server 7.1
 :description: pass:q[Couchbase Server 7.1 introduces multiple new features.]
 
+[#release-711]
+== Release 7.1.1 (July 2022)
+
+Couchbase Server 7.1.1 was released in July 2022.
+This maintenance release contains fixes to issues.
+
+=== Fixed Issues
+
+This release contains the fixes listed below.
+
+=== Data Service
+
+[#table-known-issues-711-dataservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52248[MB-52248^]
+| Memcached hangs when no password is passed for encrypted private key
+
+|===
+
+=== XDCR
+
+[#table-known-issues-711-xdcr, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51939[MB-51939^]
+| XDCR does not update memcached flag/body after txn xattribute removal if user xattr DNE
```

Comment:
For outward-facing documentation, let's replace DNE with "is not found"

Reply:
I've changed.